### PR TITLE
Allow to add ExtendMethods to Color [@types/color]

### PR DESCRIPTION
**Pull request by mokeyish**
1. definition
   ```ts
   import Color from 'color';

   declare module 'color' {
     interface ColorExtendMethods {
       toHslString(): string;
     }
   }

   Color.prototype.toHslString = function(this: Color) {
     const [h, s, l] = this.hsl().array()
     return `hsl(${h}, ${s}%, ${l}%)`
   }


   export default Color;
   export { Color };
   ```
2. usage
   ```ts
    const c1 = Color('#fff');
    const t = c1.toHslString();
   ```

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [ ] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [ ] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ ] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [ ] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [ ] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

If removing a declaration:
- [ ] If a package was never on Definitely Typed, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## color ([<kbd>unpkg</kbd>](https://unpkg.com/browse/color@latest/))
was missing the following properties:
1. The declaration doesn't match the JavaScript module 'color'. Reason:
The JavaScript module can be called or constructed, but the declaration module cannot.

The most common way to resolve this error is to use 'export =' syntax.
To learn more about 'export =' syntax, see https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require.
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 4f6be5b8efbca3700319ffa5585f9fb7a354620d
</p>

File: types/color/index.d.ts [hunk 1]
```diff
@@ -10,7 +10,9 @@ import convert = require('color-convert');
 
 type ColorParam = Color | string | ArrayLike<number> | number | { [key: string]: any };
 
-interface Color<T extends ColorParam = ColorParam> {
+export interface ColorExtendMethods { }
```
Comment (RyanCavanaugh) [inline types/color/index.d.ts:13]: This isn't needed - other packages can augment `Color` without this interface

File: types/color/tslint.json [hunk 2]
```diff
@@ -1 +1,7 @@
-{ "extends": "@definitelytyped/dtslint/dt.json" }
+{ 
+    "extends": "@definitelytyped/dtslint/dt.json",
+     "rules": {
+        "npm-naming": [true,{"mode":"code","errors":[["NeedsExportEquals",false], ["NoDefaultExport",false]]}],
+        "no-empty-interface": false
```
Comment (RyanCavanaugh) [inline types/color/tslint.json:5]: The rules are there for a reason; don't disable them

File: types/color/color-tests.ts [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import Color = require('color');
+import Color from 'color';
```
Comment (RyanCavanaugh) [inline types/color/color-tests.ts:1]: Please put this in a separate PR and cite where `color` changed its module format to justify this

**Review by RyanCavanaugh — CHANGES_REQUESTED** [review]
This PR seems largely unnecessary?

File: types/color/index.d.ts (hunk 1 above)
Comment (mokeyish) [reply types/color/index.d.ts:13]: How to augment 'Color'  without this interface ? I got the error: error TS2671: Cannot augment module 'Color'

Currently, I used [patch-package](https://github.com/ds300/patch-package) to fix this issue for my project.
